package.json

```diff
@@ -92,15 +92,16 @@
     "gulp": "^4.0.0",
     "gulp-rename": "^2.0.0",
     "gulp-sass": "^3.1.0",
+    "jsdom": "^16.4.0",
     "lodash": "^4.17.19",
     "method-override": "^2.3.5",
+    "mocha": "^6.2.2",
```

Comment:
please don't put this into `dependencies` cause this is only needed for unit test. can revert to the current state